staging/src/k8s.io/client-go/tools/cache/reflector_test.go

```diff
@@ -576,21 +576,22 @@ func TestReflectorFullListIfExpired(t *testing.T) {
 			case "0":
 				return &v1.PodList{ListMeta: metav1.ListMeta{ResourceVersion: "10"}, Items: pods[0:4]}, nil
```

Comment:
should we ensure Continue and Limit match our expectations here?

Reply:
Done.